Test/TilemapManagerTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentAssertions;
+using Hel.Tiled;
+using NUnit.Framework;
+
+namespace Test {
+
+    [TestFixture]
+    public class TilemapManagerTests {
+
+        private TilemapManager _tilemapManager;
+        
+        [SetUp]
+        public void Setup() {
+            _tilemapManager = new TilemapManager(Loader.LoadTilemap("./MapFiles/TilemapManagerTests.json"));
+        }
```

Comment:
why do you need this set up

Reply:
We set it up to not have to re-setup the `tilemapManager` in every test we run. 

Setup is also where you'd prepare any `mock` data or classes (You can look that up if you're interested, it's a way of testing stuff that has dependencies)